src/pages/vendors/components/AdditionalInfo.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @copyright Copyright (c) 2021. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @copyright Copyright (c) 2021. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { Card, Element } from '@invoiceninja/cards';
+import { SelectField, Textarea } from '@invoiceninja/forms';
+import { useStaticsQuery } from 'common/queries/statics';
+import { Alert } from 'components/Alert';
+import { useTranslation } from 'react-i18next';
+
+type Props = { data?: any; formik?: any; errors?: any };
```

Comment:
I almost never use types in the whole project, specifically not for the props. Please use interfaces.

Reply:
I am sorry about this it is due to snippets generating components with types. I will change it in the future.